_gitignore

```diff
@@ -1,3 +1,4 @@
+# TODO - get rid of unused gitignore
```

Comment:
I'm assuming this file does nothing since you have a `.gitignore` proper and you just used the `_` prefix to prevent this from firing.  If you don't need it, you can delete it and recover from source control if you ever wanted to get it back